src/test/java/com/karankumar/bookproject/backend/goal/CalculateReadingGoalTest.java

```diff
@@ -100,4 +100,12 @@ void testHowFarAheadOrBehindSchedule(){
         assertEquals(45, howFarAheadOrBehindSchedule(113,41));
         assertThat(howFarAheadOrBehindSchedule(113,86)).isZero();
     }
+
+    //Test to test the change in the method booksToReadFromStartOfYear
+    @Test
+    void testBooksToReadFromStartOfYear(){
+        int booksToReadThisYear = 104;
```

Comment:
Could you set this to `26`? This will ensure that we can test whether the return value correctly returns the right `double` (0.5)